lib/mirage_impl_misc.ml

```diff
@@ -105,3 +86,5 @@ let rec rr_iter f l =
   match l with
   | [] -> R.ok ()
   | x :: l -> f x >>= fun () -> rr_iter f l
+
+let sexp_of_fmt fmt = Fmt.kstrf Sexplib.Sexp.of_string fmt
```

Comment:
why do we need to use `Sexplib.Sexp`? I think it would be nice to not depend on sexplib if possible.